### PR TITLE
Add hashed ip for Triton

### PR DESCRIPTION
**Pull request by kookster**
For #684 
Not totally sure how to test this, but using a local script this seems to work ~, adding a single column for `byte-range`~
actually, they are fine with the separate start and end columns, they did ask for a hashed ip, so adding that instead.

*Don't merge until we get confirmation they want the truncated ipv6 used for the hashed ip*

### CODE REVIEW

**Comment by farski** [comment]
This is probably a better PR for @cavis, I've never looked at this code before. Seems reasonable to me though.

**Review by cavis — APPROVED** [review]
👍 seems good to me.

We may want to alert existing consumers of these logs to this change?  You'd hope they would be parsing the headers of the tsv.  But who knows.  Looks like you put this one last, so maybe fine.

File: etc/cdn-log-shipper/log-shipper.yml [hunk 1]
```diff
@@ -171,6 +171,15 @@ Resources:
             });
             fields.push('prx-listener-id');
 
+            datas.forEach(data => {
+              if (data['sc-range-start'].match(/^[0-9]+$/)) {
+                data['byte-range'] = `${data['sc-range-start']}-${data['sc-range-end']}`;
+              } else {
+                data['byte-range'] = '-';
```
Comment (cavis) [inline etc/cdn-log-shipper/log-shipper.yml:178]: Would we ever hit this?

Comment (kookster) [reply etc/cdn-log-shipper/log-shipper.yml:178]: Yes, it only has numeric values for a 206 response, otherwise start and end are dashes

Comment (cavis) [reply etc/cdn-log-shipper/log-shipper.yml:178]: Oh right - this is the actual request Range header field.  Vs the "byte range the server sent to the client" we use for IAB2 calculations.

**Comment by kookster** [comment]
heard back from Triton today - they don't need the byte range as a new field, they do need the ip hashed - will do that instead.

**Comment by kookster** [comment]
triton also confirmed the ipv6 should be 64 bits (truncated) before hashing, like we do for the listenerid, so this is correct

**Pull request merged**